helm/charts/determined/templates/master-deployment.yaml

```diff
@@ -23,21 +23,15 @@ spec:
       serviceAccount: determined-master-{{ .Release.Name }}
       containers:
       - name: determined-master-{{ .Release.Name }}
-        {{- if .Values.enterpriseEdition }}
-        {{- if .Values.detVersion }}
-        # detVersion is used for CI to override the appVersion.
```

Comment:
non-blocking: Keeping one copy of this comment around to explain things could be useful.

Reply:
Brought it back as a Helm comment.